WISE-Community/WISE-Client
# feat(Authoring): Allow switching between components in component info

## Changes
- Added drop down to switch between component types
- Added previous and next buttons to switch between component types
- Added component info label
- Added description label
- Added some indentation to differentiate labels from content

## Test
- Make sure the drop down switches components
- Make sure previous and next buttons switch components

Closes #1516
